src/main/java/frc/trigon/robot/subsystems/shooter/Shooter.java

```diff
@@ -83,6 +90,17 @@ public Command getPrimeShooterCommand(DoubleSupplier targetVelocity) {
                 .andThen(this::stop);
     }
 
+    public Command getPrimeShooterCommandWithDefault(DoubleSupplier targetVelocity) {
```

Comment:
rename to ` getPrimeShooterCommandWithDefaultVelocity`

Reply:
Maybe default value?

Reply:
change to IdleVelocity